openhcl/virt_mshv_vtl/src/processor/tdx/mod.rs

```diff
@@ -1762,12 +1762,7 @@ impl UhProcessor<'_, TdxBacked> {
             msr @ (hvdef::HV_X64_MSR_GUEST_OS_ID | hvdef::HV_X64_MSR_VP_INDEX) => {
                 self.backing.cvm.hv[intercepted_vtl].msr_read(msr)
             }
-            _ => self
-                .backing
-                .untrusted_synic
-                .as_mut()
-                .unwrap()
-                .read_nontimer_msr(msr),
+            _ => self.backing.untrusted_synic.as_mut().unwrap().read_msr(msr),
```

Comment:
Is it ok for these TDX paths to have access to the timer registers on the untrusted_synic?

Reply:
No.

Reply:
Will tweak